### ISSUE TITLE
[1.10.2] Crash with Sponge-Forge server when loading world

### ISSUE THREAD

**Issue by vfreex** [skyboy/MineFactoryReloaded#603]
###Versions

```
forge-1.10.2-12.18.3.2281
spongeforge-1.10.2-2281-5.2.0-BETA-2394
MineFactoryReloaded-[1.10.2]2.9.0B1-218
```

mods
```
CodeChickenLib-1.10.2-2.5.9.267-universal.jar  MineFactoryReloaded-[1.10.2]2.9.0B1-218.jar
CoFHCore-1.10.2-4.1.4.7-universal.jar          spongeforge-1.10.2-2281-5.2.0-BETA-2394.jar
```

###Server Console

```
[13:16:17] [main/INFO] [LaunchWrapper]: Loading tweak class name net.minecraftforge.fml.common.launcher
.FMLServerTweaker
[13:16:17] [main/INFO] [LaunchWrapper]: Using primary tweak class name net.minecraftforge.fml.common.la
uncher.FMLServerTweaker
[13:16:17] [main/INFO] [LaunchWrapper]: Calling tweak class net.minecraftforge.fml.common.launcher.FMLS
erverTweaker
[13:16:18] [main/INFO] [FML]: Forge Mod Loader version 12.18.3.2281 for Minecraft 1.10.2 loading
[13:16:18] [main/INFO] [FML]: Java is OpenJDK 64-Bit Server VM, version 1.8.0_131, running on Linux:amd
64:3.10.0-514.21.1.el7.x86_64, installed at /usr/lib/jvm/java-1.8.0-openjdk-1.8.0.131-3.b12.el7_3.x86_6
4/jre
[13:16:20] [main/WARN] [FML]: The coremod codechicken.lib.asm.CCLCorePlugin does not have a MCVersion a
nnotation, it may cause issues with this version of Minecraft
[13:16:20] [main/WARN] [FML]: The coremod cofh.asm.LoadingPlugin does not have a MCVersion annotation,
it may cause issues with this version of Minecraft
[13:16:20] [main/INFO] [FML]: Loading tweaker org.spongepowered.asm.launch.MixinTweaker from spongeforg
e-1.10.2-2281-5.2.0-BETA-2394.jar
[13:16:20] [main/INFO] [LaunchWrapper]: Loading tweak class name net.minecraftforge.fml.common.launcher
.FMLInjectionAndSortingTweaker
[13:16:20] [main/INFO] [LaunchWrapper]: Loading tweak class name org.spongepowered.asm.launch.MixinTwea
ker
[13:16:20] [main/INFO] [mixin]: SpongePowered MIXIN Subsystem Version=0.6.8 Source=file:/srv/docker/spo
nce-forge/mods/spongeforge-1.10.2-2281-5.2.0-BETA-2394.jar Env=SERVER
[13:16:20] [main/INFO] [mixin]: Compatibility level set to JAVA_8
[13:16:20] [main/INFO] [mixin]: Adding new token provider org.spongepowered.mod.SpongeCoremod$TokenProv
ider to MixinEnvironment[DEFAULT]
[13:16:20] [main/INFO] [mixin]: Adding new token provider org.spongepowered.mod.SpongeCoremod$TokenProv
ider to MixinEnvironment[PREINIT]
[13:16:20] [main/INFO] [mixin]: Adding new token provider org.spongepowered.mod.SpongeCoremod$TokenProv
ider to MixinEnvironment[INIT]
[13:16:20] [main/INFO] [LaunchWrapper]: Loading tweak class name net.minecraftforge.fml.common.launcher
.FMLDeobfTweaker
[13:16:20] [main/INFO] [LaunchWrapper]: Calling tweak class net.minecraftforge.fml.common.launcher.FMLI
njectionAndSortingTweaker
[13:16:20] [main/INFO] [LaunchWrapper]: Calling tweak class net.minecraftforge.fml.common.launcher.FMLI
njectionAndSortingTweaker
[13:16:20] [main/INFO] [LaunchWrapper]: Calling tweak class net.minecraftforge.fml.relauncher.CoreModMa
nager$FMLPluginWrapper
[13:16:22] [main/INFO] [FML]: Found valid fingerprint for Minecraft Forge. Certificate fingerprint e3c3
d50c7c986df74c645c0ac54639741c90a557
[13:16:22] [main/INFO] [LaunchWrapper]: Calling tweak class net.minecraftforge.fml.relauncher.CoreModMa
nager$FMLPluginWrapper
[13:16:22] [main/INFO] [LaunchWrapper]: Calling tweak class net.minecraftforge.fml.relauncher.CoreModMa
nager$FMLPluginWrapper
[13:16:22] [main/INFO] [LaunchWrapper]: Calling tweak class org.spongepowered.asm.launch.MixinTweaker
[13:16:22] [main/INFO] [mixin]: Initialised Mixin FML Remapper Adapter with net.minecraftforge.fml.comm
on.asm.transformers.deobf.FMLDeobfuscatingRemapper@665e9289
[13:16:22] [main/INFO] [LaunchWrapper]: Calling tweak class net.minecraftforge.fml.common.launcher.FMLD
eobfTweaker
[13:16:23] [main/INFO] [LaunchWrapper]: Calling tweak class net.minecraftforge.fml.relauncher.CoreModMa
nager$FMLPluginWrapper
[13:16:23] [main/INFO] [LaunchWrapper]: Loading tweak class name net.minecraftforge.fml.common.launcher
.TerminalTweaker
[13:16:23] [main/INFO] [LaunchWrapper]: Loading tweak class name org.spongepowered.asm.mixin.MixinEnvir
onment$EnvironmentStateTweaker
[13:16:23] [main/INFO] [LaunchWrapper]: Calling tweak class net.minecraftforge.fml.common.launcher.Term
inalTweaker
[13:16:23] [main/INFO] [LaunchWrapper]: Calling tweak class org.spongepowered.asm.mixin.MixinEnvironmen
t$EnvironmentStateTweaker
[13:16:25] [main/WARN] [mixin]: @Mixin target net.minecraft.server.management.PlayerProfileCache$Profil
eEntry is public in mixins.common.core.json:server.management.MixinPlayerProfileCacheEntry and should b
e specified in value
[13:16:28] [main/INFO] [LaunchWrapper]: Launching wrapped minecraft {net.minecraft.server.MinecraftServ
er}
[13:16:33] [Server thread/INFO]: Starting minecraft server version 1.10.2
[13:16:33] [Server thread/INFO] [FML]: MinecraftForge v12.18.3.2281 Initialized
[13:16:33] [Server thread/INFO] [FML]: Replaced 231 ore recipes
[13:16:35] [Server thread/WARN] [Sponge]: Skipping org.spongepowered.api.data.key.Keys.ACHIEVEMENTS
[13:16:35] [Server thread/WARN] [Sponge]: Skipping org.spongepowered.api.data.key.Keys.AGE
[13:16:35] [Server thread/WARN] [Sponge]: Skipping org.spongepowered.api.data.key.Keys.ATTACK_DAMAGE
[13:16:35] [Server thread/WARN] [Sponge]: Skipping org.spongepowered.api.data.key.Keys.BASE_SIZE
[13:16:35] [Server thread/WARN] [Sponge]: Skipping org.spongepowered.api.data.key.Keys.DAMAGE_ENTITY_MA
P
[13:16:35] [Server thread/WARN] [Sponge]: Skipping org.spongepowered.api.data.key.Keys.DOMINANT_HAND
[13:16:35] [Server thread/WARN] [Sponge]: Skipping org.spongepowered.api.data.key.Keys.FILLED
[13:16:35] [Server thread/WARN] [Sponge]: Skipping org.spongepowered.api.data.key.Keys.FLUID_LEVEL
[13:16:35] [Server thread/WARN] [Sponge]: Skipping org.spongepowered.api.data.key.Keys.HEALTH_SCALE
[13:16:35] [Server thread/WARN] [Sponge]: Skipping org.spongepowered.api.data.key.Keys.HEIGHT
[13:16:35] [Server thread/WARN] [Sponge]: Skipping org.spongepowered.api.data.key.Keys.HELD_EXPERIENCE
[13:16:35] [Server thread/WARN] [Sponge]: Skipping org.spongepowered.api.data.key.Keys.IS_SLEEPING
[13:16:35] [Server thread/WARN] [Sponge]: Skipping org.spongepowered.api.data.key.Keys.LAST_ATTACKER
[13:16:35] [Server thread/WARN] [Sponge]: Skipping org.spongepowered.api.data.key.Keys.LAST_DAMAGE
[13:16:35] [Server thread/WARN] [Sponge]: Skipping org.spongepowered.api.data.key.Keys.LEASH_HOLDER
[13:16:35] [Server thread/WARN] [Sponge]: Skipping org.spongepowered.api.data.key.Keys.PASSENGERS
[13:16:35] [Server thread/WARN] [Sponge]: Skipping org.spongepowered.api.data.key.Keys.SCALE
[13:16:35] [Server thread/WARN] [Sponge]: Skipping org.spongepowered.api.data.key.Keys.SPAWNER_ENTITIES
[13:16:35] [Server thread/WARN] [Sponge]: Skipping org.spongepowered.api.data.key.Keys.SPAWNER_MAXIMUM_
DELAY
[13:16:35] [Server thread/WARN] [Sponge]: Skipping org.spongepowered.api.data.key.Keys.SPAWNER_MAXIMUM_
NEARBY_ENTITIES
[13:16:35] [Server thread/WARN] [Sponge]: Skipping org.spongepowered.api.data.key.Keys.SPAWNER_NEXT_ENT
ITY_TO_SPAWN
[13:16:35] [Server thread/WARN] [Sponge]: Skipping org.spongepowered.api.data.key.Keys.SPAWNER_REQUIRED
_PLAYER_RANGE
[13:16:35] [Server thread/WARN] [Sponge]: Skipping org.spongepowered.api.data.key.Keys.SPAWNER_SPAWN_CO
UNT
[13:16:35] [Server thread/WARN] [Sponge]: Skipping org.spongepowered.api.data.key.Keys.SPAWNER_SPAWN_RANGE
[13:16:35] [Server thread/WARN] [Sponge]: Skipping org.spongepowered.api.data.key.Keys.STATISTICS
[13:16:35] [Server thread/WARN] [Sponge]: Skipping org.spongepowered.api.data.key.Keys.WILL_SHATTER
[13:16:35] [Server thread/WARN] [Sponge]: Skipping org.spongepowered.api.data.key.Keys.WIRE_ATTACHMENTS
[13:16:35] [Server thread/WARN] [Sponge]: Skipping org.spongepowered.api.data.key.Keys.WIRE_ATTACHMENT_EAST
[13:16:35] [Server thread/WARN] [Sponge]: Skipping org.spongepowered.api.data.key.Keys.WIRE_ATTACHMENT_NORTH
[13:16:35] [Server thread/WARN] [Sponge]: Skipping org.spongepowered.api.data.key.Keys.WIRE_ATTACHMENT_SOUTH
[13:16:35] [Server thread/WARN] [Sponge]: Skipping org.spongepowered.api.data.key.Keys.WIRE_ATTACHMENT_WEST
[13:16:35] [Server thread/WARN] [Sponge]: Skipping org.spongepowered.api.item.inventory.InventoryArchetypes.MENU_COLUMN
[13:16:35] [Server thread/WARN] [Sponge]: Skipping org.spongepowered.api.item.inventory.InventoryArchetypes.MENU_BUTTON
[13:16:35] [Server thread/WARN] [Sponge]: Skipping org.spongepowered.api.item.inventory.InventoryArchetypes.MENU_ICON
[13:16:35] [Server thread/WARN] [Sponge]: Skipping org.spongepowered.api.item.inventory.InventoryArchetypes.MENU_CHECKBOX
[13:16:35] [Server thread/WARN] [Sponge]: Skipping org.spongepowered.api.item.inventory.InventoryArchetypes.MENU_SPINNER
[13:16:37] [Server thread/INFO] [FML]: Found 0 mods from the command line. Injecting into mod discoverer
[13:16:37] [Server thread/INFO] [FML]: Searching /srv/docker/sponce-forge/mods for mods
[13:16:37] [Server thread/WARN] [FML]: ****************************************
[13:16:37] [Server thread/WARN] [FML]: * The modid CodeChickenLib is not the same as it's lowercase version. Lowercasing will be enforced i
n 1.11
[13:16:37] [Server thread/WARN] [FML]: *  at net.minecraftforge.fml.common.FMLModContainer.sanityCheckModId(FMLModContainer.java:143)
[13:16:37] [Server thread/WARN] [FML]: *  at net.minecraftforge.fml.common.FMLModContainer.<init>(FMLModContainer.java:128)
[13:16:37] [Server thread/WARN] [FML]: *  at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
[13:16:37] [Server thread/WARN] [FML]: *  at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
[13:16:37] [Server thread/WARN] [FML]: *  at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.ja
va:45)
[13:16:37] [Server thread/WARN] [FML]: *  at java.lang.reflect.Constructor.newInstance(Constructor.java:423)...
[13:16:37] [Server thread/WARN] [FML]: ****************************************
[13:16:37] [Server thread/WARN] [CodeChickenLib]: Mod CodeChickenLib is missing the required element 'version' and a version.properties fil
e could not be found. Falling back to metadata version 2.5.9.267
[13:16:37] [Server thread/INFO] [ccl-entityhook]: Mod ccl-entityhook is missing the required element 'name'. Substituting ccl-entityhook
[13:16:37] [Server thread/WARN] [ccl-entityhook]: Mod ccl-entityhook is missing the required element 'version' and no fallback can be found
. Substituting '1.0'.
[13:16:37] [Server thread/WARN] [FML]: ****************************************
[13:16:37] [Server thread/WARN] [FML]: * The modid MineFactoryReloaded|CompatVanilla is not the same as it's lowercase version. Lowercasing
 will be enforced in 1.11
[13:16:37] [Server thread/WARN] [FML]: *  at net.minecraftforge.fml.common.FMLModContainer.sanityCheckModId(FMLModContainer.java:143)
[13:16:37] [Server thread/WARN] [FML]: *  at net.minecraftforge.fml.common.FMLModContainer.<init>(FMLModContainer.java:128)
[13:16:37] [Server thread/WARN] [FML]: *  at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
[13:16:37] [Server thread/WARN] [FML]: *  at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
[13:16:37] [Server thread/WARN] [FML]: *  at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.ja
va:45)
[13:16:37] [Server thread/WARN] [FML]: *  at java.lang.reflect.Constructor.newInstance(Constructor.java:423)...
[13:16:37] [Server thread/WARN] [FML]: ****************************************
[13:16:38] [Server thread/INFO] [FML]: Forge Mod Loader has identified 11 mods to load
[13:16:38] [Server thread/INFO] [FML]: Attempting connection with missing mods [mcp, FML, Forge, spongeapi, sponge, <CoFH ASM>, CodeChicken
Lib, ccl-entityhook, cofhcore, minefactoryreloaded, MineFactoryReloaded|CompatVanilla] at CLIENT
[13:16:38] [Server thread/INFO] [FML]: Attempting connection with missing mods [mcp, FML, Forge, spongeapi, sponge, <CoFH ASM>, CodeChicken
Lib, ccl-entityhook, cofhcore, minefactoryreloaded, MineFactoryReloaded|CompatVanilla] at SERVER
[13:16:38] [Server thread/INFO] [FML]: Processing ObjectHolder annotations
[13:16:38] [Server thread/INFO] [FML]: Found 423 ObjectHolder annotations
[13:16:38] [Server thread/INFO] [FML]: Identifying ItemStackHolder annotations
[13:16:38] [Server thread/INFO] [FML]: Found 0 ItemStackHolder annotations
[13:16:38] [Server thread/INFO] [FML]: Applying holder lookups
[13:16:38] [Server thread/INFO] [FML]: Holder lookups applied
[13:16:38] [Server thread/INFO] [FML]: Applying holder lookups
[13:16:38] [Server thread/INFO] [FML]: Holder lookups applied
[13:16:38] [Server thread/INFO] [FML]: Applying holder lookups
[13:16:38] [Server thread/INFO] [FML]: Holder lookups applied
[13:16:38] [Server thread/INFO] [FML]: Configured a dormant chunk cache size of 0
[13:16:38] [Forge Version Check/INFO] [ForgeVersionCheck]: [cofhcore] Starting version check at https://raw.github.com/cofh/version/master/
cofhcore_update.json
[13:16:40] [Server thread/INFO] [CoFHWorld]: Registering default Templates...
[13:16:40] [Server thread/INFO] [CoFHWorld]: Registering default generators...
[13:16:40] [Server thread/INFO] [CoFHWorld]: Verifying or creating base world generation directory...
[13:16:40] [Server thread/INFO] [CoFHWorld]: Complete.
[13:16:41] [Forge Version Check/INFO] [ForgeVersionCheck]: [cofhcore] Found status: OUTDATED Target: 4.1.5
[13:16:41] [Forge Version Check/INFO] [ForgeVersionCheck]: [Forge] Starting version check at http://files.minecraftforge.net/maven/net/mine
craftforge/forge/promotions_slim.json
[13:16:42] [Forge Version Check/INFO] [ForgeVersionCheck]: [Forge] Found status: OUTDATED Target: 12.18.3.2316
[13:16:43] [Server thread/WARN] [FML]: ****************************************
[13:16:43] [Server thread/WARN] [FML]: * A broken ore dictionary registration with name listAllmilk has occurred. It adds an item (type: cl
ass powercrystals.minefactoryreloaded.item.ItemMFRBucketMilk) which is currently unknown to the game registry. This dictionary item can onl
y support a single value when registered with ores like this, and NO I am not going to turn this spam off. Just register your ore dictionar
y entries after the GameRegistry.
TO USERS: YES this is a BUG in the mod MineFactory Reloaded report it to them!
[13:16:43] [Server thread/WARN] [FML]: *  at net.minecraftforge.oredict.OreDictionary.registerOreImpl(OreDictionary.java:607)
[13:16:43] [Server thread/WARN] [FML]: *  at net.minecraftforge.oredict.OreDictionary.registerOre(OreDictionary.java:581)
[13:16:43] [Server thread/WARN] [FML]: *  at net.minecraftforge.oredict.OreDictionary.registerOre(OreDictionary.java:579)
[13:16:43] [Server thread/WARN] [FML]: *  at powercrystals.minefactoryreloaded.setup.recipe.Vanilla.registerOredict(Vanilla.java:135)
[13:16:43] [Server thread/WARN] [FML]: *  at powercrystals.minefactoryreloaded.MineFactoryReloadedCore.preInit(MineFactoryReloadedCore.java
:128)
[13:16:43] [Server thread/WARN] [FML]: *  at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)...
[13:16:43] [Server thread/WARN] [FML]: ****************************************
[13:16:43] [Server thread/INFO] [FML]: Applying holder lookups
[13:16:43] [Server thread/INFO] [FML]: Holder lookups applied
[13:16:43] [Server thread/INFO] [FML]: Injecting itemstacks
[13:16:43] [Server thread/INFO] [FML]: Itemstack injection complete
[13:16:43] [Server thread/INFO]: Loading properties
[13:16:43] [Server thread/INFO]: Default game type: SURVIVAL
[13:16:43] [Server thread/INFO]: Generating keypair
[13:16:43] [Server thread/INFO]: Starting Minecraft server on 127.0.0.1:25572
[13:16:43] [Server thread/INFO]: Using epoll channel type
[13:16:43] [Server thread/WARN]: **** SERVER IS RUNNING IN OFFLINE/INSECURE MODE!
[13:16:43] [Server thread/WARN]: The server will make no attempt to authenticate usernames. Beware.
[13:16:43] [Server thread/WARN]: While this makes the game possible to play without internet access, it also opens up the ability for hacke
rs to connect with any username they choose.
[13:16:43] [Server thread/WARN]: To change this, set "online-mode" to "true" in the server.properties file.
[13:16:44] [Server thread/INFO] [FML]: Injecting itemstacks
[13:16:44] [Server thread/INFO] [FML]: Itemstack injection complete
[13:16:44] [Server thread/INFO] [CoFHWorld]: Found 4 World Generation files and 0 folders present in /srv/docker/sponce-forge/config/cofh/w
orld.
[13:16:44] [Server thread/INFO] [CoFHWorld]: Unmet dependencies to load cofh/world/thermalfoundation_oil.json
[13:16:44] [Server thread/INFO] [CoFHWorld]: Unmet dependencies to load cofh/world/thermalfoundation_ores.json
[13:16:44] [Server thread/INFO] [CoFHWorld]: Unmet dependencies to load cofh/world/thermalfoundation_clathrates.json
[13:16:44] [Server thread/INFO] [cofhcore]: CoFH Core: Load Complete.
[13:16:44] [Server thread/INFO] [minefactoryreloaded]: Load Complete.
[13:16:44] [Server thread/INFO] [FML]: Forge Mod Loader has successfully loaded 11 mods
[13:16:44] [Server thread/INFO]: Preparing level "ggo00"
[13:16:44] [Server thread/INFO] [Sponge]: Checking for worlds that need to be migrated...
[13:16:44] [Server thread/INFO] [Sponge]: No worlds were found in need of migration.
[13:16:44] [Server thread/INFO] [FML]: Injecting existing block and item data into this server instance
[13:16:44] [Server thread/ERROR]: Exception reading ./ggo00/level.dat
java.lang.NullPointerException
        at org.spongepowered.mod.SpongeMod.lambda$new$1(SpongeMod.java:169) ~[SpongeMod.class:1.10.2-2281-5.2.0-BETA-2394]
        at org.spongepowered.mod.registry.SpongeGameData.lambda$addRegistryCallback$0(SpongeGameData.java:55) ~[SpongeGameData.class:1.10.2
-2281-5.2.0-BETA-2394]
        at net.minecraftforge.fml.common.registry.FMLControlledNamespacedRegistry.addObjectRaw(FMLControlledNamespacedRegistry.java:584) ~[
FMLControlledNamespacedRegistry.class:?]
        at net.minecraftforge.fml.common.registry.FMLControlledNamespacedRegistry.add(FMLControlledNamespacedRegistry.java:482) ~[FMLContro
lledNamespacedRegistry.class:?]
        at net.minecraftforge.fml.common.registry.FMLControlledNamespacedRegistry.loadIds(FMLControlledNamespacedRegistry.java:776) ~[FMLCo
ntrolledNamespacedRegistry.class:?]
        at net.minecraftforge.fml.common.registry.PersistentRegistryManager.loadPersistentDataToStagingRegistry(PersistentRegistryManager.j
ava:460) ~[PersistentRegistryManager.class:?]
        at net.minecraftforge.fml.common.registry.PersistentRegistryManager.injectSnapshot(PersistentRegistryManager.java:280) ~[Persistent
RegistryManager.class:?]
        at net.minecraftforge.fml.common.FMLContainer.readData(FMLContainer.java:327) ~[FMLContainer.class:?]
        at net.minecraftforge.fml.common.FMLCommonHandler.handleWorldDataLoad(FMLCommonHandler.java:418) ~[FMLCommonHandler.class:?]
        at net.minecraft.world.storage.SaveFormatOld.loadAndFix(SaveFormatOld.java:118) [azz.class:?]
        at net.minecraft.world.storage.SaveHandler.redirect$onLoadWorldInfo$zph000(SaveHandler.java:1064) [azy.class:?]
        at net.minecraft.world.storage.SaveHandler.func_75757_d(SaveHandler.java:121) [azy.class:?]
        at org.spongepowered.common.world.WorldManager.loadAllWorlds(WorldManager.java:707) [WorldManager.class:1.10.2-2281-5.2.0-BETA-2394
]
        at net.minecraft.server.MinecraftServer.func_71247_a(MinecraftServer.java:3489) [MinecraftServer.class:?]
        at net.minecraft.server.dedicated.DedicatedServer.func_71197_b(DedicatedServer.java:266) [ld.class:?]
        at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:431) [MinecraftServer.class:?]
        at java.lang.Thread.run(Thread.java:748) [?:1.8.0_131]
[13:16:44] [Server thread/WARN] [FML]: Forge Mod Loader detected that the backup level.dat is being used.

This may happen due to a bug or corruption, continuing can damage
your world beyond repair or lose data / progress.

It's recommended to create a world backup before continuing.

Run the command /fml confirm or or /fml cancel to proceed.
Alternatively start the server with -Dfml.queryResult=confirm or -Dfml.queryResult=cancel to preselect the answer.

```

It doesn't occur when creating the world at the very first time. But after restarting the server, it occurs.

**Comment by skyboy** [comment]
Does this issue persist in build 220?

**Comment by vfreex** [comment]
After upgrading to MFR build 220 and CoFHCore-1.10.2-4.1.6.10, it seems to me this issue has been fixed.
Thank you so much.

**Comment by skyboy** [comment]
Fixed by 1e2b55ad087830c0c073c8a40b296e3c0fd20983
